### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
- Remove all rights, add copyrights
- Add space at bottom
- Remove minutes and seconds from create dao
- Update dao to DAO in DAO table
- Add margin in wallet widget between wallet type and address
- add information tooltip for locked and available token
- Use info tooltip in create dao

### CODE REVIEW

File: src/components/AppHeader.vue [hunk 1]
```diff
@@ -77,12 +77,25 @@
...
+									available
+									<info-tool-tip
+										data="The tokens are available for you to deposit and vote."
+									/>
+								</template>
+								{{ availableTokens }}</a-descriptions-item
+							>
+							<a-descriptions-item>
+								<template #label>
+									*locked
```
Comment (vuvoth) [inline src/components/AppHeader.vue]:
```suggestion
									locked
```

File: src/components/AppHeader.vue [hunk 2]
```diff
@@ -77,12 +77,25 @@
...
+									<info-tool-tip
+										data="The tokens are locked for the voting period. You can
+												withdraw them once the voting period is finished."
```
Comment (rmeena840) [inline src/components/AppHeader.vue]:
```suggestion
												withdraw them once the voting period ends."
```

File: src/components/InfoToolTip.vue [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<template>
+	<a-popover>
+		<template #content>
+			<p>
+				{{ data }}
+			</p>
+		</template>
+		<info-circle-outlined style="color: #1890ff" />
```
Comment (rmeena840) [inline src/components/InfoToolTip.vue]: We should move color to something like color.constant. The benefit of this is that, we have to change at one place if we decide to change the color all over the app.

File: src/pages/CreateDao.vue [hunk 4]
```diff
@@ -22,75 +22,88 @@
...
 				<a-form-item
-					label="Gov Token ID"
 					name="token_id"
 					:rules="[{ required: true, type: 'number' }]"
-					extra="ASA which will define Gov DAO token membership."
 				>
+					<template #label>
+						<div class="margin_right_extra_sm">Gov Token ID</div>
+						<info-tool-tip
+							data="ASA which will define Gov DAO token membership."
```
Comment (rmeena840) [inline src/pages/CreateDao.vue]:
```suggestion
							data="ASA that will define Gov DAO token membership."
```

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviewed

File: src/components/InfoToolTip.vue (hunk 3 above)
Comment (Megha-Dev-19) [reply src/components/InfoToolTip.vue]: yeah, added

File: src/constants/constant.ts [hunk 5]
```diff
@@ -78,3 +78,5 @@ export const DAO_CONTRACT_STATE_CONFIG = {
 	globalInts: 5,
 	globalBytes: 2,
 };
+
```
Comment (rmeena840) [inline src/constants/constant.ts]:
```suggestion

// Color constants
```

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviewed

**Review by rmeena840 — APPROVED** [review]
lgtm!!

**Pull request merged**